backend/exchanges/pocket.go

```diff
@@ -145,3 +145,47 @@ func PocketWidgetSignAddress(account accounts.Interface, message string, format
 
 	return addr.EncodeForHumans(), base64.StdEncoding.EncodeToString(sig), nil
 }
+
+// PocketWidgetVerifyAddress allows the user to verify an address for the Pocket Iframe workflow.
```

Comment:
please document the return value. it seems they are error codes? why not return them as `error` if that is the case?

Reply:
As suggested in the other comment, there was no need of return values, now it only returns error